src/pages/about/index.jsx

```diff
@@ -2,34 +2,41 @@ import React from "react";
 import Member from "./Member";
 import portraitItems from "./Data";
 import { AboutContainer, Cards, Pledge, PledgeBox, Partners } from "./style";
+import Title from "../../components/Title/index";
 
 const About = () => {
   return (
     <AboutContainer>
       <Pledge>
-        <h1>Notre Engagement</h1>
+        <Title titleName={"Notre Engagements"} />
         <PledgeBox>
           <div>
             <p>
               <span>Notre mission : accompagner</span> les jeunes dans leur
               orientation professionnelle, <span>révèler</span> leurs talents
               pour leur permettre d’être
-              <span>libres, responsables et heureux</span>.
+              <span> libres, responsables et heureux</span>.
             </p>
             <p>
-              Ce qui nous anime : c’est permettre aux jeunes une première belle
-              rencontre avec le monde de l’entreprise qui leur donne envie.
-            </p>
-            <p>
-              Leur permettre de croire que <span>tout est possible</span>.
+              <ul>
+                <span>Ce qui nous anime :</span>
+                <li>
+                  Permettre aux jeunes une première belle rencontre avec le
+                  monde de l’entreprise qui leur donne envie.
+                </li>
+                <li className="fas fa-angle-right">
+                  Leur permettre de croire que
+                  <span> tout est possible</span>.
+                </li>
+              </ul>
             </p>
           </div>
           <img src="/images/atelierbis.jpg" alt="atelier" />
         </PledgeBox>
       </Pledge>
 
-      <div>
-        <h1>L'Equipe de Toi Demain</h1>
+      <div id="equipe">
+        <Title titleName={"L'Equipe de Toi Demain"} />
```

Comment:
Curly braces are not needed on strings: `<Title titleName="L'Equipe de Toi Demain" />`